src/index.js

```diff
@@ -66,7 +70,7 @@ window.addEventListener('load', async() => {
     store.dispatch(actions.connectWeb3(window.web3))
     window.web3.eth.getAccounts(function (error, accounts) {
       let player;
-      if(accounts.length !== 0 && !error) player = accounts[0]
+      if (accounts.length !== 0 && !error) player = accounts[0]
```

Comment:
Pls don't make code style changes that are just that -code style changes-.

Reply:
Noted